app/controllers/airportDetailsController.js

```diff
@@ -0,0 +1,18 @@
+const airportDetailDao = require('../dao/airportDetailDao');
+const routes = require('express').Router();
+const logger = require('../util/logger').createLogger('airportDetailController');
+const cbFunction = require('./controllerCallBack');
+
+routes.get('/airportDetails', function(req, res) {
+    airportDetailDao.get(req.query, function(error, result) {
+        cbFunction(error, result, res, logger);
```

Comment:
You *could* extract this callback (shared with the other route below) to a named function that both routes could call, but since it's only one line it's probably not worth it.